.github/workflows/ci.yaml

```diff
@@ -318,7 +333,7 @@ jobs:
       - name: Restore executable flag
         run: chmod a+x ./target/debug/*
       - name: Generate Golem OpenAPI yaml
-        working-directory: golem-service
+        working-directory: golem-public-api
         run: ../target/debug/golem-service-yaml > ../target/golem-service.yaml
```

Comment:
this needs to be fixes
there will be two

```
run: ../target/debug/golem-template-service-yaml > ../target/golem-template-service.yaml
```


```
run: ../target/debug/golem-worker-service-yaml > ../target/golem-worker-service.yaml
```

Reply:
No need of it for now

Reply:
It works now because it’s merged to be golem-service-yaml. Intentional it works in CI. Not accidental.

Reply:
ok